### ISSUE TITLE
Can't open attribute (can't locate attribute: 'l_max') when running timeSOAP.ipynb

### ISSUE THREAD

**Issue by phys-chem** [GMPavanLab/SOAPify#84]
**Describe the bug**
KeyError                                  Traceback (most recent call last)
/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb Cell 7 line 2
     [24](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=23)             slide = 1
     [26](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=25)         return nAt, timedSOAP, np.diff(timedSOAP.T, axis=-1)
---> [29](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=28) nAtoms, tSOAP, dtSOAP = getTimeSOAP(soapFileName, trajAddress)

/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb Cell 7 line 6
      [4](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=3) print(f)
      [5](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=4) ds = f[f"/SOAP/{trajAddress}"]
----> [6](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=5) fillSettings = getSOAPSettings(ds)
      [7](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=6) print(fillSettings)
      [8](vscode-notebook-cell://ssh-remote%2B10.36.96.214/nfs/scistore14//timesoap/SOAPify/Examples/timeSOAP.ipynb#W6sdnNjb2RlLXJlbW90ZQ%3D%3D?line=7) print(ds.shape)

File [~/software/soapify/lib/python3.10/site-packages/SOAPify/utils.py:345](https://vscode-remote+ssh-002dremote-002b10-002e36-002e96-002e214.vscode-resource.vscode-cdn.net/nfs/scistore14/chenggrp/jichen/timesoap/SOAPify/Examples/~/software/soapify/lib/python3.10/site-packages/SOAPify/utils.py:345), in getSOAPSettings(fitsetData)
    328 """Gets the settings of the SOAP calculation
    329 
    330     you can feed directly this output to :func:`fillSOAPVectorFromdscribe`
   (...)
    342 
    343 """
    344 print(fitsetData)
--> 345 lmax = fitsetData.attrs["l_max"]
    346 nmax = fitsetData.attrs["n_max"]
    347 symbols, atomicSlices = getSlicesFromAttrs(fitsetData.attrs)
...
File h5py/_objects.pyx:55, in h5py._objects.with_phil.wrapper()

File h5py/h5a.pyx:80, in h5py.h5a.open()

KeyError: "Can't open attribute (can't locate attribute: 'l_max')"

**Additional context**
I use "wget https://github.com/GMPavanLab/dynNP/releases/download/V1.0-trajectories/ico309.hdf5" and put the data file in the same directory with "trajAddress = './' .

**Comment by Iximiel** [comment]
Oh, that file was generated with an older version, without attributes, or with attributes with a different name,I don't remember well.

Without any context I can only guess that you are using the latest version of SOAPify.

To reproduce the calculations of that repository your environment should use the right [requirements.txt](https://github.com/GMPavanLab/dynNP/blob/Pubblication/requirements.txt) (from v0.0.3, the version used for that paper, to v0.1 a lot of things have changed)

**Comment by phys-chem** [comment]
> Oh, that file was generated with an older version, without attributes, or with attributes with a different name,I don't remember well.
> 
> Without any context I can only guess that you are using the latest version of SOAPify.
> 
> To reproduce the calculations of that repository your environment should use the right [requirements.txt](https://github.com/GMPavanLab/dynNP/blob/Pubblication/requirements.txt) (from v0.0.3, the version used for that paper, to v0.1 a lot of things have changed)

Thank you for your help! I am using the latest version, is there an example for the latest version?

**Comment by Iximiel** [comment]
Yes, look [here](https://github.com/GMPavanLab/SOAPify/tree/main/Examples), I hope this helps
